status/dockerfile

```
@@ -0,0 +1,14 @@
+# filepath: /Users/vineet/Projects/sce_system_status/prom/Dockerfile.scraper
+FROM python:3.9-slim
+
+WORKDIR /app
+
+COPY ./status/requirements.txt .
+
+RUN pip3 install -r requirements.txt
+
+COPY ./status/flags.py .
+COPY ./status/scraper.py .
+COPY ./status/health.html /app/health.html
+
+ENTRYPOINT ["python3", "scraper.py"]
```

make this file called `Dockerfile`, capital D
```suggestion
COPY ./status/flags.py .

COPY ./status/scraper.py .

COPY ./status/health.html /app/health.html

ENTRYPOINT ["python3", "scraper.py"]

```